TestAssets/TestPackages/dotnet-dependency-context-test/dotnet-dependency-context-test.csproj

```diff
@@ -18,7 +18,7 @@
       <Version>1.0.3</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.DependencyModel">
-      <Version>1.0.1-beta-000933</Version>
+      <Version>1.1.0</Version>
```

Comment:
Should we introduce this version into `Microsoft.DotNet.Cli.DependencyVersions.props` and remove all this duplication?